src/in.c

```diff
@@ -363,6 +374,12 @@ ly_in_skip(struct ly_in *in, size_t count)
         return LY_EDENIED;
     }
 
+    for (size_t i = 0; i < count; i++) {
+        if (in->current[i] == '\n') {
+            LY_IN_NEW_LINE(in);
+        }
+    }
+
```

Comment:
Not sure, but the performance problem may be present here as well.

Reply:
yeah, the same as in `ly_in_read()`. I'm also not sure if it is a performance issue, but I don't like this part either.